lib/registries/addon/overview/route.ts

```diff
@@ -76,23 +79,68 @@ export default class Overview extends GuidRoute {
                 ),
                 institution: (institutions as SparseModel[]).map(institution => institution.name as string),
             };
-            const headTags = [...this.metaTags.getHeadTags(metaTagsData)];
+
+            const metaTags: HeadTagDef[] = this.metaTags.getHeadTags(metaTagsData);
+
+            // Google Structured Data
+            let jsonLD: object | undefined;
+            let scriptTag;
+            try {
+                jsonLD = await this.returnStructuredData(id);
+                const jsonString: object | undefined = jsonLD ? jsonLD : undefined;
+                if (jsonString) {
+                    scriptTag = {
+                        type: 'script',
+                        content: jsonString,
+                        attrs: {
+                            type: 'application/ld+json',
+                        },
+                    };
+                } else {
+                    scriptTag = undefined;
+                }
+                if (scriptTag) {
+                    metaTags.push(scriptTag);
+                }
+            } catch (e) {
+                const errorMessage = this.intl.t('general.structured_data.json_ld_retrieval_error');
+                captureException(e, { errorMessage });
+            }
+
```

Comment:
```suggestion
    try {
            const jsonLD = await this.returnStructuredData(id);
            const scriptTag = {
                type: 'script',
                content: JSON.stringify(jsonLD),
                attrs: {
                    type: 'application/ld+json',
                },
            };
            metaTags.push(scriptTag);
        } catch (e) {
            const errorMessage = this.intl.t('general.structured_data.json_ld_retrieval_error');
            captureException(e, { errorMessage });
        }
```

Two things:

1. We need to `stringify` the jsonLD object before putting it into the DOM.
2. In JS, `undefined` is the default state of all variables when it is not initialized. Assigning `undefined` to variables is an anti-pattern.

Reply:
Ok, this has been updated. Currently testing.